### PR TITLE
Initial wg-auth repo setup

### PR DESCRIPTION
**Pull request by sudo-bmitch**
Here's the initial setup for the working group. This mirrors a fair bit from wg-reference-types, but strips out details that don't apply.

### CODE REVIEW

**Review by mikebrow — APPROVED** [review]
LGTM

**Review by jcarter3 — APPROVED** [review]

File: GOVERNANCE.md
Comment (toddysm) [inline GOVERNANCE.md:23]: Just curious... is there any guidance how to unblock? For example, super majority rule or overwhelming vote from the community (whatever that means). Should we add something to the governance that clarifies that?

Comment (sudo-bmitch) [reply GOVERNANCE.md:23]: I copied that from the reference types working group. I'd be comfortable with a majority of maintainers resolving any conflict. I'm not too concerned since our output still needs to be approved by other existing OCI groups (TOB or an individual project like distribution-spec, and then another vote for the change to be released).

**Pull request merged**
